src/main/java/black/door/node/java/benchmark/BenchTestJersey.java

```diff
@@ -0,0 +1,54 @@
+package black.door.node.java.benchmark;
+
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.*;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.IntStream;
+
+/**
+ * Created by cjbur on 11/15/2015.
+ */
+@Path("/")
+public class BenchTestJersey {
+
+    @GET
+    @Path("/fib/{param}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response fib(@PathParam("param") int n) {
+        if (n > 0) {
```

Comment:
just needs one thing
should sleep for a number of ms given in a path parameter "sleep"